src/stories/Forms/Select/styled.ts

```diff
@@ -0,0 +1,158 @@
+import styled from 'styled-components';
+import rem from '../../../shared/pxToRem';
+import { ChevronDown } from '@huddly/frokost/havre';
+
+export const Wrapper = styled.div`
+  position: relative;
+
+  figure,
+  img,
+  svg {
+    width: 100%;
+    max-width: var(--spacing-32);
+    max-height: var(--spacing-32);
+    object-fit: contain;
+  }
+`;
+
+export const SelectButton = styled.button<{
+  isOpen: boolean;
+  hasLabel: boolean;
+  hasError: boolean;
+}>`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: var(--spacing-48);
+  padding: var(--spacing-16) var(--spacing-8) var(--spacing-16) var(--spacing-16);
+  border: ${(p) => (p.hasError ? 'var(--border-error)' : 'var(--border-primary)')};
+  border-radius: ${rem(3)};
+  color: ${(p) => (p.hasLabel ? 'var(--color-grey15)' : 'var(--color-grey55)')};
+  cursor: pointer;
+  background-color: ${(p) => (p.isOpen ? 'var(--color-grey96)' : 'var(--color-grey99)')};
+  column-gap: var(--spacing-8);
+  width: 100%;
+
+  &:hover {
+    background-color: var(--color-grey96);
+  }
+
+  &:focus,
+  &:focus-within {
+    border: var(--border-active);
+    outline: 0;
+    background-color: var(--color-grey96);
+  }
+`;
+
+/**
+ * The selected content inside the SelectButton.
+ */
+export const SelectedContent = styled.span<{ hasNestedContent: boolean }>`
+  display: block;
+
+  ${(p) =>
+    p.hasNestedContent &&
+    `
+      display: flex;
+      align-items: center;
+      column-gap: var(--spacing-8);
+
+    `}
+
+  &,
+    & > * {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+`;
+
+export const RotatingChevron = styled(ChevronDown)<{ rotate: boolean }>`
```

Comment:
Are you sure this is not a carrot? 🐇

Reply:
[maybe](https://www.gingersoftware.com/english-online/spelling-book/confusing-words/carat-caret-karat-carrot)